Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Driver.hs

```diff
@@ -863,14 +864,19 @@ data DriverMode
 type FleetTotalEarningAPI =
   "fleet"
     :> "totalEarning"
+    :> QueryParam "from" UTCTime
+    :> QueryParam "to" UTCTime
     :> Get '[JSON] FleetTotalEarningResponse
 
 type FleetVehicleEarningAPI =
   "fleet"
     :> "vehicleEarning"
-    :> Capture "vehicleNo" Text
-    :> QueryParam "driverId" (Id Driver)
-    :> Get '[JSON] FleetEarningRes
+    :> QueryParam "vehicleNo" Text
+    :> QueryParam "limit" Int
+    :> QueryParam "offset" Int
+    :> QueryParam "from" UTCTime
+    :> QueryParam "to" UTCTime
+    :> Get '[JSON] FleetEarningListRes
```

Comment:
should we handle backward compatibility here?

Reply:
These api are not used by ui currently.